assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -115,13 +115,22 @@ function Title({
   canCollapse = true,
 }) {
   const {
-    state: { isCollapsed, height, resizeable, panelContentId },
-    actions: { collapse, expand, setHeight, setExpandToHeight, resetHeight },
+    state: { isCollapsed, height, resizeable, panelContentId, panelTitleId },
+    actions: {
+      collapse,
+      expand,
+      setHeight,
+      setExpandToHeight,
+      resetHeight,
+      confirmTitle,
+    },
   } = useContext(panelContext);
   const {
     state: { inspectorContentHeight },
   } = useInspector();
 
+  useEffect(confirmTitle, [confirmTitle]);
```

Comment:
Not all panels have a title ("No selection" panel in particular), and if a panel doesn't have a title but still has `aria-labelledby` referencing a non-existent id, AXE e2e tests start failing as it's an illegal WAI-ARIA property.

This is not the prettiest way to do it, but it works.